custom_components/frigate/config_flow.py

```diff
@@ -147,6 +149,13 @@ async def async_step_init(
                     True,
                 ),
             ): bool,
+            vol.Optional(
+                CONF_EXPIRE_NOTIFICATIONS_AFTER_MINS,
+                default=self._config_entry.options.get(
+                    CONF_EXPIRE_NOTIFICATIONS_AFTER_MINS,
+                    0,
```

Comment:
I think I'll argue for a non-zero (i.e. not never expire) default, despite the fact that's the current behavior. Maybe 1-day to limit the exposure (however minuscule) users have. How many notifications are used after 1-day?

Reply:
That sounds ok to me. I will default this to 1440

Reply:
This would be a change in behavior from before my involvement in this project, so lets let @blakeblackshear weigh-in here on whether or unauthenticated notification accesses expiring after 1-day feels okay to them.

Reply:
I would err on the side of caution and not change default behavior, but I will set this to whatever value we deem appropriate